ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -54,9 +54,17 @@ namespace ngraph
                             splits[0].get_node_shared_ptr(),
                             num_groups_const,
                             std::make_shared<default_opset::Divide>(splits[1], num_groups_const)};
```

Comment:
Can you try Gather to grab the dimensions from ShapeOf instead of Split?

Reply:
Or use OutputVector instead of NodeVector and remove `splits[0].get_node_shared_ptr()`

Reply:
Thanks! Done